twilio/rest/resources/base.py

```diff
@@ -377,6 +378,7 @@ def count(self):
         resp, page = self.request("GET", self.uri)
         return page["total"]
 
+    @normalize_dates
```

Comment:
This probably needs to get applied to the NextGenListResource implementation as well. Check that the normalize_dates functionality is correct for next-gen formats while doing so.